From 339e583efa3b0ea35518a9d0c7fc11309c2d0efd Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Mon, 12 May 2025 14:42:56 +0300
Subject: [PATCH] [SW-221409] gaudi3: add serdes backup access addr

Loading of backup serdes partition image to die1 ARC,
requires definition of accessing this address from ARC.

Change-Id: Ic657967225e4c5f4c9e3b03dddca02cbabbcea62
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
---

diff --git a/gaudi3/gaudi3_mem_addr_dt.h b/gaudi3/gaudi3_mem_addr_dt.h
index db780b8..afd7361 100644
--- a/gaudi3/gaudi3_mem_addr_dt.h
+++ b/gaudi3/gaudi3_mem_addr_dt.h
@@ -328,6 +328,7 @@
 #define ARC_PID_RSVD_DRAM_SIZE				0x1000000
 
 /* SerDes backup in Die1 Arc1 DCCM */
+#define D1_ARC1_DCCM_OFFSET				0x2B080000
 #define D1_ARC1_DCCM_HI					0x0300007F
 #define D1_ARC1_DCCM_LO					0xDB080000
 #define D1_ARC1_DCCM_SIZE				0x40
